index.html

```diff
@@ -9,18 +9,22 @@
   </head>
   <body>
     <div class="container">
-      <div class="task1">
+      <div class="task-box task1">
         <p>Task 1</p>
         <ul class="task1-list"></ul>
       </div>
-      <div class="task2">
+      <div class="task-box task2">
         <p>Task 2</p>
         <ul class="task2-list"></ul>
       </div>
-      <div class="task3">
+      <div class="task-box task3">
```

Comment:
We're not insisting on BEM for this project but keep it in mind when authoring markup